code/modules/mob/living/carbon/xenomorph/castes/King.dm

```diff
@@ -0,0 +1,103 @@
+/datum/caste_datum/king
+	caste_type = XENO_CASTE_KING
+	caste_desc = "The end of the line."
+	tier = 4
+
+	melee_damage_lower = XENO_DAMAGE_TIER_6
+	melee_damage_upper = XENO_DAMAGE_TIER_8
+	melee_vehicle_damage = XENO_DAMAGE_TIER_5
+	max_health = XENO_HEALTH_KING
+	plasma_gain = XENO_PLASMA_GAIN_TIER_3
+	plasma_max = XENO_PLASMA_TIER_10
+	xeno_explosion_resistance = XENO_EXPLOSIVE_ARMOR_TIER_7
+	armor_deflection = XENO_ARMOR_FACTOR_TIER_5
+	speed = XENO_SPEED_TIER_1
+
+	minimum_evolve_time = XENO_KING_ACQUISITION_TIME // Just for the notification
+	evolves_to = null
+	deevolves_to = null
+	evolution_allowed = FALSE
+	can_vent_crawl = FALSE
+
+	behavior_delegate_type = /datum/behavior_delegate/king_base
+
+	tackle_min = 6
+	tackle_max = 10
+
+	minimap_icon = "xenoqueen"
+
+	fire_immunity = FIRE_IMMUNITY_NO_DAMAGE
+
+/mob/living/carbon/xenomorph/king
+	caste_type = XENO_CASTE_KING
+	name = XENO_CASTE_KING
+	desc = "A massive alien covered in spines and armoured plates."
+	icon = 'icons/mob/xenos/king.dmi'
+	icon_size = 64
+	icon_state = "King Walking"
+	plasma_types = list(PLASMA_CHITIN)
+	pixel_x = -16
+	old_x = -16
+	mob_size = MOB_SIZE_IMMOBILE
+	tier = 4
+	small_explosives_stun = FALSE
+	counts_for_slots = FALSE
+
+	claw_type = CLAW_TYPE_VERY_SHARP
+	age = -1
+	aura_strength = 6
```

Comment:
consider adding organ_value now that dissections are merged. 8k is queen.

Reply:
abom is 20k...

Reply:
This is never getting killed so I am gonna add an absurd value
However this does need sprites

Reply:
the organ sprite? you can use the t3 sprite probably. get_organ_icon proc does that